site/website/blog/2020-04-01-upgrading-to-jet-40.md

```diff
@@ -158,8 +158,8 @@ We made multiple breaking changes in Jet’s own APIs too:
   `withUnorderedAsyncResponses()` were removed from `ServiceFactory`.
   These properties are relevant only in the context of asynchronous
   operations and were used in conjunction with
-  `GeneralStage.mapUsingServiceAsync(…​)`. In Jet 4.0 the
-  `GeneralStage.mapUsingServiceAsync(…​)` method has a new variant with
+  `GeneralStage.mapUsingServiceAsync(…)`. In Jet 4.0 the
+  `GeneralStage.mapUsingServiceAsync(…)` method has a new variant with
```

Comment:
What's the diff here? Just curious :)

Reply:
The old version includes this unicode character: [ZERO WIDTH SPACE](https://www.fileformat.info/info/unicode/char/200b/index.htm). It is visible when opening the file with Intellij.